src/clanhr/ring_test_client/core.clj

```diff
@@ -5,6 +5,12 @@
   (:use clojure.test
         ring.mock.request))
 
+(defn ->edn [raw]
```

Comment:
does this function should called `<from-format>->edn` instead of `->edn` ? I don't think this name is much idiomatic.